lib/docparser.js

```diff
@@ -191,7 +192,8 @@ YUI.add('docparser', function (Y) {
             'parma': 'param', // typo
             'propery': 'property', // typo
             'prop': 'property', // probably meant property
-            'returns': 'return' // need to standardize on one or the other
+            'returns': 'return', // need to standardize on one or the other
+            'throw': 'throws' // need to standardize on one or the other
```

Comment:
I don't think we need `throw` or `exception` unless users tell us it's a common mistake.